src/game/stage/grind_effect_2.c

```diff
@@ -48,93 +55,157 @@ struct Task *sub_801F15C(s16 x, s16 y, u8 param2, s8 param3, TaskMain main, Task
 
     return t;
 }
+#endif
 
+#if (GAME == GAME_SA1)
+void sa2__Task_801F214(void)
```

Comment:
Not a fan of this. Like I said, my main concern is that if we are trying to "share" things, it should  be because they are actually the same thing. This grind effect has changed so much between the 2 games that sharing this file is a nightmare.

Reply:
If we merged this repos, I'd be much happier with a copy of this existing twice.